internal/rbd/clone.go

```diff
@@ -181,6 +181,16 @@ func (rv *rbdVolume) createCloneFromImage(ctx context.Context, parentVol *rbdVol
 		return err
 	}
 
+	// resize the image to the requested size is greater than the current size
+	if rv.VolSize > parentVol.VolSize {
```

Comment:
repetitive code blocks, really should be done with a helper function

Reply:
should be fixed now